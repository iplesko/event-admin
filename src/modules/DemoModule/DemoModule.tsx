import React from 'react';
import './style.scss';

export default (): React.ReactElement => (
    <p className="test">abcdefg</p>
)
