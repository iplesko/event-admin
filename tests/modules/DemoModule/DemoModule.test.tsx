import React from 'react'
import { render } from "@testing-library/react";
import DemoModule from "../../../src/modules/DemoModule/DemoModule";

describe('DemoModule', () => {
  test("should render correctly", () => {
    const { baseElement } = render(<DemoModule/>);

    expect(baseElement).toMatchSnapshot();
  })
})
