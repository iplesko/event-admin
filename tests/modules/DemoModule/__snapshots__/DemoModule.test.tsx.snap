// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DemoModule should render correctly 1`] = `
<body>
  <div>
    <p
      class="test"
    >
      abcdefg
    </p>
  </div>
</body>
`;
